examples/drill-downs/dashboard-app/.env.development

```diff
@@ -1,2 +1,4 @@
 REACT_APP_CUBEJS_TOKEN=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpYXQiOjE1OTI4NzEzODJ9.QF1G45jmqjMmJBRygcYKRICSSmIXk48AEDXgAEv0U-Q
 REACT_APP_API_URL=http://localhost:4000/cubejs-api/v1
+CUBE_EXAMPLES_MENU_TOKEN=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpYXQiOjE2NDY0ODI4NDl9.xxO5HFPrXMnTRFbF5vyNkpAgJGx04h-ZOJA-7O9uiIE
```

Comment:
Since we routinely use `CUBEJS_API_SECRET=SECRET`, it would make sense to use this long-lived token everywhere: https://github.com/cube-js/cube.js/blob/master/examples/recipes/non-additivity/queries/run.sh#L8

Reply:
Documented it here: https://www.notion.so/Best-Practices-WIP-fb86c44df16740dfab22bb0fe19e7d98#2fd206cde7d4444fa108e67b852f6af6